src/syscheckd/db/include/fimDB.hpp

```diff
@@ -13,17 +13,19 @@
 #define _FIMDB_HPP
 #include "dbsync.hpp"
 #include "rsync.hpp"
-#include "fimCommonDefs.h"
 #include <condition_variable>
 #include <mutex>
 
 #ifdef __cplusplus
 extern "C"
 {
-#include "logging_helper.h"
+#include "fimCommonDefs.h"
 }
 #endif
 
+const auto FIM_COMPONENT_FILE {"fim_file"};
```

Comment:
This definition can be written as `#define FIM_COMPONENT_FILE "fim_file"` what would be defined on compile time instead of runtime

Reply:
done